### DISCUSSION TITLE
Contribution reg.

### DISCUSSION THREAD

**Discussion by nivramam** [ManimCommunity/manim#1961] [General]
Hello there, this is Niveydhithaa, a final year student at PSG College of Technology (Coimbatore, India). As part of my course "Open Source Software Lab", I would like to understand and contribute to this manim repo as I am interested in this domain. 

Since I am a beginner to contributions, I just had a clarification as to whether is there any procedure to begin my contributions journey and do I need to send an e-mail to the admins? Thanks a lot in advance, and waiting to get started!

**Comment by huguesdevimeux** [comment]
Hello there, and welcome to the open source world ! 


No, you don’t have to message anyone. The most simple thing is just to open a PR about something, and then people (likely manim’s devs, but sometimes others) will review it. then It can be merged.

You can see a list of things that a beginner could do in the library by sorting the issues with the label « good first issue ».

You may also want to join the discord server linked in the README. 

Have fun !

**Reply by nivramam** [reply]
Sure, will make a note of these while I get started. Thanks a lot

**Comment by Darylgolden** [comment]
Hello and thank you for your interest in contributing to Manim! It would be helpful if you could read our [contributing guidelines](https://docs.manim.community/en/latest/contributing.html). While not required, you're strongly encouraged to join our [Discord server](https://www.manim.community/discord/index.html) as that's where most discussions occur. Let us know if you need anymore help! Happy contributing!

**Reply by nivramam** [reply]
Oh fine, that's great. Thanks a lot
